### DISCUSSION TITLE
Racknerd：美国西海岸SanJose(圣何塞)VPS优惠促销，低至$11.88/年

### DISCUSSION THREAD

**Discussion by zhuziyi1989** [zhuziyi1989/tools#5] [Ideas]
Racknerd 在本月(2022年4月) 单独针对美国西海岸sanjose（圣何塞）机房搞了优惠促销，3款特别便宜的vps，年付低至$11.88/年（续费约人民币70元）。racknerd 有着比较牛逼的售后以及靠谱的技术支持，加之性价比高，比较适合预算偏低的用户。

**性价比极高，经本人长期访问测试（洛杉矶），国内中国移动的支持最稳定，电信和联通次之，稳定性较高，如果仅用于查查资料完全够用。**

🌏 官方网站：[https://www.racknerd.com](https://my.racknerd.com/aff.php?aff=2440)

💴 支持付款方式：PayPal、支付宝、信用卡、加密货币、webmoney等

**SanJose VPS促销款**

KVM虚拟，纯SSD raid10，1Gbps带宽，solusvm面板管理，自带一个IPv4

| **内存** | **CPU** | **SSD** | **流量** | **价格**  | **购买**                                                   |
| -------- | ------- | ------- | -------- | --------- | ---------------------------------------------------------- |
| 1G       | 1核     | 12G     | 1T/月    | $11.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=648)** |
| 2G       | 1核     | 25G     | 3T/月    | $17.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=649)** |
| 3G       | 2核     | 55G     | 5T/月    | $29.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=650)** |

**以下是历史促销，官方已隐藏但仍然可购买**

多个机房可供选择，站长把同配置里面价格最低的给大家挑选出来了，有兴趣的围观下！

| **内存** | **CPU** | **SSD** | **流量** | **价格**  | **购买**                                                   |
| -------- | ------- | ------- | -------- | --------- | ---------------------------------------------------------- |
| 512M     | 1核     | 10G     | 1T/月    | $9.89/年  | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=620)** |
| 768M     | 1核     | 13G     | 1T/月    | $10.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=632)** |
| 1G       | 1核     | 20G     | 2T/月    | $13.98/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=621)** |
| 1.5G     | 1核     | 23G     | 3T/月    | $15.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=633)** |
| 2G       | 2核     | 23G     | 3.5T/月  | $20.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=74)**  |
| 2G       | 2核     | 35G     | 5T/月    | $22.49/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=513)** |
| 2.5G     | 2核     | 45G     | 5T/月    | $21.5/年  | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=477)** |
| 2.5G     | 3核     | 25G     | 5T/月    | $22.00/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=171)** |
| 2.5G     | 3核     | 35G     | 6T/月    | $23.50/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=98)**  |
| 3G       | 3核     | 45G     | 6.5T/月  | $24.28/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=199)** |
| 3.5G     | 3核     | 45G     | 7T/月    | $28.99/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=136)** |
| 4G       | 3核     | 45G     | 6T/月    | $35.68/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=153)** |
| 4G       | 3核     | 45G     | 6T/月    | $35.68/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=76)**  |
| 4G       | 4核     | 55G     | 8T/月    | $39.89/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=195)** |
| 5.5G     | 4核     | 88G     | 10T/月   | $63.88/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=254)** |
| 8G       | 4核     | 100G    | 5T/月    | $89.00/年 | **[链接](https://my.racknerd.com/aff.php?aff=2440&pid=312)** |

**Comment by linzeennnn** [comment]
很奇怪,从25年12月开始,Racknerd的机房似乎全被中国移动墙了,典型的ping正常,tcp,udp全部极高延迟(几乎不可用)

**Reply by zhuziyi1989** [reply]
因为那是特殊时期，现在都恢复正常了。

**Comment by mimohua** [comment]
牛逼，全都涨价了
